### PR TITLE
MarkdownBear: Add prerequisite check

### PR DESCRIPTION
**Pull request by Adrianzatreanu**
Fixes https://github.com/coala-analyzer/coala-bears/issues/586

### CODE REVIEW

File: bears/markdown/MarkdownBear.py [hunk 1]
```diff
@@ -7,7 +7,10 @@
 @linter(executable='remark',
         use_stdin=True,
         output_format='corrected',
-        result_message='The text does not comply to the set style.')
+        result_message='The text does not comply to the set style.',
+        prerequisite_check_command=('remark-cli'),
+        prerequisite_check_fail_message='remark-cli is not installed. Please'
+        'make sure you install ``remark-cli`` and not ``remark``.')
```
Comment (mr-karan) [inline bears/markdown/MarkdownBear.py]: might want to add a link to the tool i.e. https://www.npmjs.com/package/remark-cli ?

Comment (sils1297) [inline bears/markdown/MarkdownBear.py]: I don't think so, this is fine.

File: bears/markdown/MarkdownBear.py [hunk 2]
```diff
@@ -7,7 +7,10 @@
 @linter(executable='remark',
         use_stdin=True,
         output_format='corrected',
-        result_message='The text does not comply to the set style.')
+        result_message='The text does not comply to the set style.',
+        prerequisite_check_command=('remark-cli'),
```
Comment (sils1297) [inline bears/markdown/MarkdownBear.py]:
```
lasse@lsx1 ~/prog/coala (git)-[sils/usab] % remark
Error: No input
    at Ware.stdin (/usr/lib/node_modules/remark-cli/node_modules/unified-engine/lib/file-set-pipeline/stdin.js:51:14)
    at Ware.<anonymous> (/usr/lib/node_modules/remark-cli/node_modules/wrap-fn/index.js:45:19)
    at next (/usr/lib/node_modules/remark-cli/node_modules/ware/lib/index.js:85:20)
    at /usr/lib/node_modules/remark-cli/node_modules/wrap-fn/index.js:121:18
    at /usr/lib/node_modules/remark-cli/node_modules/unified-engine/lib/file-set-pipeline/file-system.js:81:13
    at /usr/lib/node_modules/remark-cli/node_modules/unified-engine/lib/finder.js:192:13
    at /usr/lib/node_modules/remark-cli/node_modules/vfile-find-down/index.js:285:9
    at next (/usr/lib/node_modules/remark-cli/node_modules/vfile-find-down/index.js:242:13)
    at visitAll (/usr/lib/node_modules/remark-cli/node_modules/vfile-find-down/index.js:253:5)
    at find (/usr/lib/node_modules/remark-cli/node_modules/vfile-find-down/index.js:284:12)
lasse@lsx1 ~/prog/coala (git)-[sils/usab] % remark-cli
zsh: command not found: remark-cli
```

does this work?!

**Comment by sils1297** [comment]
as discussed

**Pull request closed without merging**